source/installation/windows-no-admin.md

```diff
@@ -41,12 +41,14 @@ A version number should be printed, and no error message should appear. Congrats
 
 ## 3. Install OpenFisca
 
+For this example, we will utilise the `OpenFisca-France` package. For a different model you could achieve the same result by following the [offline environment](offline-environment) example to get local copies of the required packages.
```

Comment:
Our goal is to only refer to the Country Template, not to any specific country package.

Reply:
Thank you for the review and feedback. I note that this specific example for windows no administration needs to be retried  utilising the latest country template. It was originally written against the France model and I would like to leave it in that state for accuracy until there's time to test and write an updated solution.